### ISSUE TITLE
[Cypher] Non meaningful error when omitting colon before relationship type

### ISSUE THREAD

**Issue by ikwattro** [neo4j/neo4j#6568]
Minor issue but would help beginners to have a more meaningful error here : 

Query : 

```
create (a)-[ASSOCIATED_WITH]->(b);
```

```
Error: A single relationship type must be specified for CREATE (line 1, column 11 (offset: 10))
"create (a)-[ASSOCIATED_WITH]->(b);"
```

**Comment by ikwattro** [comment]
Update 3.2, same error.

**Comment by Lojjs** [comment]
Thanks for your suggestion and sorry for the late replay. I  totally agree that the error message is not very clear for a beginner which easy makes this mistake. We will look into this.

Best regards Cypher Team through Louise

**Comment by Lojjs** [comment]
@ikwattro The error message has now been clarified and will be different depending on if you forget the colon or try to add multiple labels (which the old error message implied). This fix will be included in the next patch releases of version 3.1 - 3.3
